maps/CdaToBundle.4.map

```diff
@@ -1047,6 +1047,99 @@ group CdaPatientRoleToFhirPatient(source cda_patientRole : PatientRole, target f
+            extension.value = translate(qualifier,'#ELGAEntityNamePartQualifierFHIRNamePartQualifier','code' );
+        };
+        cda_name_item_family -> fhir_patient_name_ad.family = (cda_name_item_family.xmlText);
+      };
+  };
+};
+  // For all other family elements, all elements are included.
+  cda_patient_name.item as cda_name_item then {
+    cda_name_item.family as cda_name_item_family where (qualifier.contains('BR').not() and qualifier.contains('AD').not() ) then {
+      cda_name_item_family -> 
+        fhir_patient.name = create('HumanName') as fhir_humanName then { 
+
+          // name.use
+          cda_name_item.use as cda_name_item_use -> fhir_humanName.use = translate(cda_name_item_use, '#ELGAEntityNameUseFHIRNameUse', 'code');
+
+          // name.text
+          cda_name_item.xmlText as cda_name_item_xmlText -> fhir_humanName.text = (cda_name_item_xmlText);
+
+          // name.prefix
+          cda_name_item.prefix as cda_name_item_prefix where ( qualifier.where($this = 'BR').exists().not() and qualifier.where($this = 'AD').exists().not()) then{
+            cda_name_item_prefix -> fhir_humanName.prefix = (cda_name_item_prefix.xmlText) as fhir_humanName_prefix then {
+              cda_name_item_prefix.qualifier as qualifier ->
+                fhir_humanName_prefix.extension as extension,
+                extension.url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier',
+                extension.value = translate(qualifier, '#ELGAEntityNamePartQualifierFHIRNamePartQualifier', 'code');
+              };
+          };
+            
+          // name.given
+          cda_name_item.given as cda_name_item_given where (qualifier.where($this = 'BR').exists().not() and qualifier.where($this = 'AD').exists().not()) then {
+            cda_name_item_given -> fhir_humanName.given = (cda_name_item_given.xmlText) as fhir_humanName_given then {
+              cda_name_item_given.qualifier as qualifier ->
+                fhir_humanName_given.extension as extension,
+                extension.url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier',
+                extension.value = translate(qualifier, '#ELGAEntityNamePartQualifierFHIRNamePartQualifier', 'code');
+            };
+          };
+          
+          //name.family
+            cda_name_item_family -> fhir_humanName.family = (cda_name_item_family.xmlText) as fhir_humanName_family then {
+              cda_name_item_family.qualifier as qualifier ->
+                fhir_humanName_family.extension as extension,
+                extension.url = 'http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier',
+                extension.value = translate(qualifier, '#ELGAEntityNamePartQualifierFHIRNamePartQualifier', 'code');
+            };
+          
+          // name.suffix
+          cda_name_item.suffix as cda_name_item_suffix where (qualifier.where($this = 'BR').exists().not() and qualifier.where($this = 'AD').exists().not()) then {
```

Comment:
If we exclude suffixes having qualifier="BR" or qualifier="AD" here, where do we map the suffixes having these qualifiers?